scripts/common/install-latest-librdkafka.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#### Install latest librdkafka, right from the GIT repo.
+
+### Define the version of kafka we are pinning to.
+kafka_version=1.5.0
+
+
+### Install librdkafka
+## Install dependencies
+apk add --no-cache libressl-dev || exit 2
+apk add --no-cache cyrus-sasl-dev || exit 2
+apk add --no-cache zstd-dev || exit 2
+apk add --no-cache zlib-dev || exit 2
+apk add --no-cache git || exit 2
+apk add --no-cache make || exit 2
+apk add --no-cache gcc || exit 2
+apk add --no-cache g++ || exit 2
+
+## Fetch librdkafka from github
+git clone https://github.com/edenhill/librdkafka.git || exit 2
+cd librdkafka/ || exit 2
+
+## Check out the version we plan to build, so it's pinned.
+git checkout v${kafka_version} || exit 2
+
+## Actually build and install librdkafka
+./configure --install-deps || exit 2
```

Comment:
Does `--install-deps` only put in runtime dependencies?

Reply:
It depends on what you mean by "runtime". If you mean that it installs everything needed for a developer to be able to compile a program that uses exotic features of librdkafka, then yes.